docs/users.rst

```diff
@@ -743,6 +743,10 @@ pkg_managers
   environment variables, dependencies, or extra configuration will be provided
   with the sources.
 
+  The full information about supported package managers is in the
+  `Cachito documentation: package managers
```

Comment:
Looks nice! I have just a nitpick:

Can we say "upstream Cachito package manager documentation" here and make this string the URL for the repo?